packages/@n8n/task-runner/package.json

```diff
@@ -42,6 +42,7 @@
     "acorn-walk": "8.3.4",
     "lodash": "catalog:",
     "luxon": "catalog:",
+    "moment": "2.30.1",
```

Comment:
Now with `moment` on `NODE_FUNCTION_ALLOW_EXTERNAL` something needs to install it so that it's included on the js task runners node_modules - this was the simplies way to accomplish this that I could think of 😅

Reply:
Sadly this means a user who doesn't want this cannot easily remove it. Let's revisit later on.

Reply:
Yep, but lets ship this now with the extra dep added to task-runner so that this works?
I feel like we'd want some extra package.json + lock for the external dependencies, same for the python runner